AutomatedTesting/Gem/PythonTests/Atom/CMakeLists.txt

```diff
@@ -8,9 +8,48 @@
 
 if(PAL_TRAIT_BUILD_HOST_TOOLS AND PAL_TRAIT_BUILD_TESTS_SUPPORTED)
     ly_add_pytest(
-        NAME AutomatedTesting::Atom_TestSuite_Main
+        NAME AutomatedTesting::Atom_Main_Null_Render_00
         TEST_SUITE main
-        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Main.py
+        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Main_Null_Render_Level_Loads.py
+        TEST_SERIAL
+        TIMEOUT 600
+        RUNTIME_DEPENDENCIES
+            AssetProcessor
+            AutomatedTesting.Assets
+            Editor
+        COMPONENT
+            Atom
+    )
+    ly_add_pytest(
+        NAME AutomatedTesting::Atom_Main_Null_Render_01
+        TEST_SUITE main
+        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Main_Null_Render_Component_01.py
```

Comment:
nitpick: maybe rename this and the other `PATH` files with "Components" instead of "Component" since its multiple (non-blocking feedback)